README.md

```diff
@@ -1,25 +1,41 @@
-# DISCLAIMER
+# Charmed OpenStack Upgrader
 
-This is a work in progress prototype. The code contained in this repository
-may not be representative of what the final approach should be.
-It is likely that the end result will live under a different name, in a
-different repository, and only use some of the code and ideas found here.
+Charmed OpenStack Upgrader (COU) is an application (packaged as a snap) to upgrade
+a Canonical distribution of [Charmed OpenStack](https://ubuntu.com/openstack/docs/overview)
+in an automated and frictionless manner. The application detects the version of the
+running cloud and proposes an upgrade plan to the next available OpenStack release.
+
+For more information, please refer to [COU Documentation](https://charmed-openstack-upgrader.readthedocs-hosted.com/).
 
 # Setup
 
+The Charmed OpenStack Upgrader snap can be installed directly from the snap store:
+
 ```bash
-# Instructions for local builds until we have automatic connections and alias
-make clean
-sudo snap remove charmed-openstack-upgrader --purge
-make build
-sudo snap install ./charmed-openstack-upgrader.snap --dangerous
-sudo snap connect charmed-openstack-upgrader:juju-client-observe snapd
-sudo snap connect charmed-openstack-upgrader:dot-local-share-cou snapd
-sudo snap connect charmed-openstack-upgrader:ssh-public-keys snapd
-sudo snap alias charmed-openstack-upgrader.cou cou
```

Comment:
This can be part of a `CONTRIBUTING.md`of the project. What do you think? Maybe not in this PR, but in a future one. It might worth open a new issue to track it.

Reply:
Sure. I opened an [issue](https://github.com/canonical/charmed-openstack-upgrader/issues/211) to track adding `CONTRIBUTING.md`